nupic/simple_server.py

```diff
@@ -0,0 +1,134 @@
+#! /usr/bin/env python
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2013, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+"""
+A simple web server for interacting with NuPIC.
+Note: Requires web.py to run (install using '$ pip install web.py')
+"""
+import datetime
+import json
+import web
+
+from nupic.frameworks.opf.modelfactory import ModelFactory
+
+
+
+g_models = {}
```

Comment:
if https://github.com/numenta/nupic/pull/1929 gets merged, this can be simplified